auth0/resource_auth0_client.go

```diff
@@ -887,24 +911,28 @@ func clientHasChange(c *management.Client) bool {
 }
 
 func flattenCustomSocialConfiguration(customSocial *management.ClientNativeSocialLogin) []interface{} {
-	if customSocial != nil {
-		m := make(map[string]interface{})
+	if customSocial == nil {
+		return nil
+	}
```

Comment:
We return early in case `customSocial` is nil. This decreases the level of indentation within this func.